db/migrate/20240402051116_add_delegate_roles.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddDelegateRoles < ActiveRecord::Migration[7.1]
+  def change
+    User.where.not(delegate_status: nil).each do |user|
+      UserRole.create!(
+        user_id: user.id,
+        group_id: user.read_attribute(:region_id),
+        start_date: '2004-08-01',
```

Comment:
I'm assuming you will manually backfill this once the migration is done?

Reply:
Yes correct.